onnxruntime/python/tools/kernel_explorer/kernels/gemm.cc

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <type_traits>
+
+#include <pybind11/pybind11.h>
+
+#include "core/providers/rocm/rocm_common.h"
+#include "device_array.h"
+#include "operator.h"
+
+#include "core/providers/rocm/shared_inc/fpgeneric.h"
+
+namespace py = pybind11;
+
+namespace onnxruntime {
+
+enum class BlasOp {
+  N,
+  T,
+  // C,
+};
+
+// assume row majored
+template <typename T>
+class GemmBase : public Operator {
+ public:
+  GemmBase(
+      BlasOp opa, BlasOp opb,
+      int64_t m, int64_t n, int64_t k,
+      double alpha,
+      DeviceArray& a, int64_t lda,
+      DeviceArray& b, int64_t ldb,
+      double beta,
+      DeviceArray& c, int64_t ldc)
```

Comment:
There may not be a best parameter order here, but how about this order: a, b, c, m, n, k, alpha, beta, opa, opb?

Reply:
FYI, this is the same parameter order as in cublas and rocblas.

Reply:
Thanks. This sounds good to me.